addons/web/static/src/core/debug/profiling/profiling_service.js

```diff
@@ -46,9 +46,13 @@ const profilingService = {
                 params
             );
             const resp = await orm.call("ir.profile", "set_profiling", [], kwargs);
-            state.session = resp.session;
-            state.collectors = resp.collectors;
-            state.params = resp.params;
+            if (resp.type == "ir.actions.act_window") {
```

Comment:
Maybe
```suggestion
            if (resp.type) {
```
instead so it's possible to e.g. neuter the endpoint?